src/site/content/en/spaces/2022-08-04-ui-ama/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Ask me anything with the UI DevRel Team
+description: Join the Chrome UI DevRel Team to ask your questions about building modern web interfaces.
+hosts:
+  - bramus
+  - jhey
+  - una
+primary_host:
+  - una
+event_date: 2022-08-04
+event_time: 9AM PT
+cal_link: "https://calendar.google.com/calendar/u/0/r/eventedit?&text=Ask+me+anything+with+the+UI+DevRel+Team&details=Join+the+Chrome+UI+DevRel+Team+to+ask+your+questions+about+building+modern+web+interfaces.&location=Twitter+Spaces&dates=20220804T160000Z%2F20220804T170000Z"
+audio: tbd
```

Comment:
Hardcoding `"tbd"` for the audio URL isn't _great_.

Maybe leave it blank, and have the template only conditionally output an `<audio>` element if the property is set?